src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp

```diff
@@ -169,14 +183,36 @@ std::vector<CPUSpecificParams> memForm4D_dynamic = {
     CPUSpecificParams({nChw16c}, {nChw16c}, {}, "unknown")
 };
 
+const ov::AnyMap emptyConfig = {};
+const ov::AnyMap i64Config = {{PluginConfigInternalParams::KEY_CPU_NATIVE_I64, PluginConfigParams::YES}};
```

Comment:
Can we add i64 native config (and probably not native, with KEY_CPU_NATIVE_I64=NO) to CPUTestUtils?